package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Displays a progress in a circle",
   "scripts": {
     "start": "grunt",
-    "checkTypes": "tsc --noEmit",
+    "typeCheck": "tsc --noEmit",
```

Comment:
Pre commit, should also be renamed

Reply:
Fixed. Strange because I was able to commit...

Reply:
missing npm dependency `pre-commit`

Reply:
Now it all makes sense